### DISCUSSION TITLE
play store installs aves on external storage

### DISCUSSION THREAD

**Discussion by idontveapc** [deckerst/aves#796] [General]
seriously asking, is it ok?

**Comment by deckerst** [comment]
As an app developer, i configured the app so it can be installed on either kind of storage. But as a user you decide where it goes. You should be able to change that in the OS app settings.

![Screenshot_20231018_085929_Settings](https://github.com/deckerst/aves/assets/13898333/0674e111-753f-4a82-a5ba-7189563683ed)

**Reply by idontveapc** [reply]
yeah, but is it good on external or internal?

**Reply by idontveapc** [reply]
would play store keep updating if i move it on internal?

**Reply by deckerst** [reply]
I don't think it matters much. I believe the option to install it on the SD card used to be useful especially for people with little internal storage.

As for updates, it will most likely work fine, no worries.
